src/components/Game/Result/Result.js

```diff
@@ -1,36 +1,118 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { Redirect } from 'react-router-dom';
 import { connect } from 'react-redux';
+import classNames from 'classnames';
 
-const ResultView = ({ words, answer }) => {
-    const isAnswerCorrect = () => words.includes(answer);
-
-    return (
-        <div>
-            {
-                (isAnswerCorrect())
-                    ? 'gratulacje, wygrałeś'
-                    : (
-                        `nie udało się, może następnym razem.
-                        poprawne słowa to:
-                        ${words
-                            .map((word) => word.toUpperCase())
-                            .join(', ')}`
-                    )
-            }
-        </div>
-    );
-};
+import classes from './Result.scss';
+import { submitAnswer } from '../../../store/actions/answer';
+
+class ResultView extends Component {
+    componentWillUnmount() {
+        const { dispatch } = this.props;
+        dispatch(submitAnswer(''));
+    }
+
+    isAnswerCorrect = () => {
+        const { words, answer } = this.props;
+        return words.includes(answer);
+    };
+
+    handlePlayAgainClick = () => {
+        const { history } = this.props;
+        history.replace('/game');
+    };
+
+    render() {
```

Comment:
this `render` is way tooo long, it has to be simplified, shortened, maybe subcomponents should be extracted

Reply:
👍